lib/sitespeed-utils.js

```diff
@@ -1,7 +1,12 @@
 var Sitespeed = require('sitespeed.io/lib/sitespeed');
-var chalk = require('chalk');
-var format = require('string-format');
- 
+var common = require('./common')
+var Mocha = require('mocha');
+var fs = require('fs');
+var path = require('path');
+budgetTestcasePath = '../config/budget-testcases.json';
```

Comment:
Ok... well I see in the next commit that you removed the line, so disregard my comment. :)